english/33hours/explore-outside/outside.md

```diff
@@ -5,4 +5,5 @@ left and your right. You stand there several minutes, but see no cars at all.
 The motel parking lot is completely empty. It looks to be abandoned. There are no other buildings around for as far as
 you can see.
 
-[Go back inside and explore the motel room.](../explore-room/room.md)
\ No newline at end of file
+[Go back inside and explore the motel room.](../explore-room/room.md)
+[Start walking east, on the highway](./explore-east.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.